packages/forma-36-react-components/src/components/Autocomplete/Autocomplete.tsx

```diff
@@ -89,12 +97,18 @@ const reducer = (state: State, action: Action): State => {
     case QUERY_CHANGED:
       return {
         ...state,
-        highlightedItemIndex: null,
+        highlightedItemIndex: action.multiselect
+          ? state.highlightedItemIndex
+          : null,
```

Comment:
It seems to me here that you want the state, more specifically the `highlightedItemIndex`, to stay the same if the multiselect is true... is that right?

If that's so, what do you think about doing something like this?

```js
{
  ...state,
   // only if multiselect is false you alter the state
  ...(!action.multiselect && { highlightedItemIndex: null }),
  isOpen: true,
  query: action.payload
}
```